### DISCUSSION TITLE
wiki needs a blacklist

### DISCUSSION THREAD

**Discussion by chrislusf** [community/community#27733] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to keep the wiki open to any users who found wiki problems.

However, there are some bad users starts to add malicious links to the wiki. I can revoke the changes after reviewing. But sometimes they keep coming back.

For these users, I want to manually put them in a short blacklist.

**Comment by airtower-luna** [comment, marked as answer]
According to the documentation blocking a user also prevents them from editing wiki pages on your repositories, so maybe you can just do that? I assume someone who spams malicious links is not someone you interact with anyway. :sweat_smile: 

See: https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account

**Reply by chrislusf** [reply]
Unluckily I just moved the repo to an organization.

**Reply by airtower-luna** [reply]
Fortunately the same applies to organization repositories, there's just a separate documentation page for how it works: https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-organization

**Reply by chrislusf** [reply]
Thanks a lot! This does help!

**Comment by spenserblack** [comment]
What do you think about the reverse? I think it would be cool to have a wiki-only collaborators list. In other words a whitelist instead of a blacklist.

**Reply by chrislusf** [reply]
this requires adding people to collaborators, which is not many users would want to do to fix some typos. I want to avoid this step.

**Comment by cdefgah** [comment]
If your repository wiki is being changed by users who post spam/malicious links, you can restrict the access to your wiki only to collaborators.

Check this official doc: https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis

**Reply by chrislusf** [reply]
this requires adding people to collaborators, which is not many users would want to do to fix some typos. I want to avoid this step.

**Comment by jfullstackdev** [comment]
yup that's right you have two options either block users that are doing that or restrict it to collaborators only

**Reply by chrislusf** [reply]
Right. That is my point: wiki needs a blacklist.
